qcelemental/data/psi4_dft_info.py

```diff
@@ -0,0 +1,5773 @@
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "blyp-d3mbj": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "blyp-d3m(bj)": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "blyp-nl": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": True,
+            "deriv": 1,
+        },
+        "bop": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "bop-d3zero": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "bop-d3": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "bop-d3bj": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "bop-d3(bj)": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "b86bpbe": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "pw86pbe": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "pbe": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "pbe-d2": {
+            "ansatz": 1,
+            "c_hybrid": False,
+            "x_hybrid": False,
+            "c_lrc": False,
+            "x_lrc": False,
+            "nlc": False,
+            "deriv": 1,
+        },
+        "pbe-d": {
+            "
```

Comment:
I don't understand what the three mp2 ones are.

Reply:
they're passing off mp2 and mp2-d (mp2 + empirical dispersion) as DFT functionals so that the latter can use the dispersion infrastructure that's naturally attached to DFT. sort of like how HF is a DFT functional only a bit more egregious.

I doubt they'll hurt anything -- just didn't want to fool you that everything was straight from libxc.

Reply:
Oh I see that now. I guess it's okay to keep them in there. In the end, I suppose MP2 is just a double-hybrid...

Reply:
This logic we keep us in good standing with `hf-3c` and like methods in here.